includes/malloc.h

```diff
@@ -0,0 +1,31 @@
+#ifndef MALLOC_H
+#define MALLOC_H
+
+#include <stdint.h>
+#include <unistd.h>
+
+extern uint32_t _ebss;
+
+#define HEAP_START ((void*)&_ebss);
+
+#define ALIGN(x) (((((x) - 1) >> 2) << 2) + 4)
+
+#define HEADER_SIZE (sizeof(size_t) * 2 + sizeof(int) + sizeof(void*) * 2)
+
+#define RAM_SIZE 32000
+
+typedef struct  block_header_s
+{
+    size_t                  size;
+    size_t                  used;
+    struct block_header_s   *next;
+    struct block_header_s   *prev;
+    int                     free;
+    char                    data[1];
```

Comment:
What is this data? I think the proper way is to use `uint8_t data[]` (“flexible array member”)